### PR TITLE
[accounting] Added command to deal with unterminated radacct entries #47

### PR DESCRIPTION
**Pull request by R9295**
Implements n. 3 of #47

### CODE REVIEW

File: django_freeradius/management/commands/cleanup_stale_radacct.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from django.utils.timezone import now
+
+from django_freeradius.models import RadiusAccounting
+
+
+class Command(BaseCommand):
+    help = "Closes active accounting sessions older than <days>"
+
+    def add_arguments(self, parser):
+        parser.add_argument('number_of_days', type=int)
```
Comment (nemesisdesign) [inline django_freeradius/management/commands/cleanup_stale_radacct.py]: could you add a default value of 15 days?

File: django_freeradius/tests/test_admin.py [hunk 2]
```diff
@@ -213,3 +213,15 @@ def test_nas_admin_save_model(self):
         self.assertNotContains(response, 'error')
         nas.refresh_from_db()
         self.assertEqual(nas.type, 'my-custom-type')
+
+    def test_cleanup_stale_radacct_command(self):
+        RadiusAccounting.objects.create(
+            unique_id='117', username='bob', nas_ip_address='127.0.0.1', start_time='2017-06-10 10:50:00',
+            authentication='RADIUS', connection_info_start='f', connection_info_stop='hgh',
+            input_octets='1', output_octets='4', update_time='2017-03-10 11:50:00'
```
Comment (nemesisdesign) [inline django_freeradius/tests/test_admin.py]: I suppose some of these attributes are shared between the other tests that you created previously, if that's the case please consider creating a dictionary which is reused, eg:

```python
options = _RADACCT.copy()
options['username'] = 'example-of-how-to-change-an-attribute'
RadiusAccounting.objects.create(**options)
```

File: django_freeradius/tests/test_admin.py [hunk 3]
```diff
@@ -213,3 +213,15 @@ def test_nas_admin_save_model(self):
         self.assertNotContains(response, 'error')
         nas.refresh_from_db()
         self.assertEqual(nas.type, 'my-custom-type')
+
+    def test_cleanup_stale_radacct_command(self):
```
Comment (nemesisdesign) [inline django_freeradius/tests/test_admin.py]: I only realized now the tests for the management commands shouldn't go in `test_admin.py`. Please move all the tests you created recently to a new file named `test_commands.py`.
You should do this change in a separate commit than the current one but you can leave both in the current PR

File: docs/source/general/management_commands.rst [hunk 4]
```diff
@@ -42,3 +42,18 @@ For example:
 .. code-block:: shell
 
     ./manage.py delete_old_postauth 365
+
+``cleanup_stale_radacct``
+-------------------------
+
+This command closes active RADIUS sessions that are open since <days>.
```
Comment (nemesisdesign) [inline docs/source/general/management_commands.rst]: Please improve it as follows:

`This command closes stale RADIUS sessions that have remained open for the number of specified `<days>`.`

File: docs/source/general/management_commands.rst [hunk 5]
```diff
@@ -42,3 +42,18 @@ For example:
...
+
+.. code-block:: shell
+
+    ./manage.py cleanup_stale_radacct <days>
+
+For example:
+
+.. code-block:: shell
+
+    ./manage.py cleanup_stale_radacct 30
```
Comment (nemesisdesign) [inline docs/source/general/management_commands.rst]: let's use 15 because is a more realistic value

File: django_freeradius/tests/test_admin.py (hunk 3 above)
Comment (peppelinux) [reply django_freeradius/tests/test_admin.py]: https://github.com/openwisp/django-freeradius/issues/35

**Comment by nemesisdesign** [comment]
@R9295 tell it to skip tests for the sample app (the app which is needed to test the swappable models), aftr this issue if you could help us to fix #35 it would be really useful because this problem is slowing us down a lot now (we have hit it at least 3-4 times in 1 week), we have GCI task ready for that, @peppelinux just created it - I only have to review it and publish it.
If something is not clear ask us on chat.

**Comment by R9295** [comment]
@nemesisdesign gotcha
